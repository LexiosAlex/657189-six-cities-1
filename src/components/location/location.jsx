import React from "react";
import PropTypes from "prop-types";

export const Location = (props) => {
  const {loc} = props;
  return <li className="locations__item">
    <a className={`${loc.active ? `tabs__item--active` : ``} locations__item-link tabs__item`} href="#">
      <span>{loc.city}</span>
    </a>
  </li>;
};

Location.propTypes = {
  loc: PropTypes.object.isRequired,
};
