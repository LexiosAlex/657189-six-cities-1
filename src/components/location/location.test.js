import React from 'react';
import renderer from 'react-test-renderer';
import {Location} from './location.jsx';

const cityLocation = {
  city: `Paris`
};

it(`Render Location`, () => {
  const tree = renderer
    .create(<Location
      loc={cityLocation}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
