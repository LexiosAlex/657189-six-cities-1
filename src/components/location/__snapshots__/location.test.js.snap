// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Location 1`] = `
<li
  className="locations__item"
  onClick={[Function]}
>
  <a
    className="tabs__item--active locations__item-link tabs__item"
    href="#"
  >
    <span>
      Amsterdam
    </span>
  </a>
</li>
`;
